dulwich/refs.py

```diff
@@ -403,8 +403,9 @@ def get_peeled(self, name):
 class DiskRefsContainer(RefsContainer):
     """Refs container that reads refs from disk."""
 
-    def __init__(self, path):
+    def __init__(self, path, **kwargs):
```

Comment:
There is no need to use **kwargs here. You can just use 'worktree_path=None', then use 'worktree_path or path' below.